src/design-tokens/ThemeProvider.tsx

```diff
@@ -1,13 +1,26 @@
-import React from 'react';
+import React, { useState } from 'react';
 import { ThemeProvider as MuiThemeProvider } from '@material-ui/core/styles';
 import { ThemeProvider as EmotionThemeProvider } from 'emotion-theming';
 
-import theme from './theme';
+import ThemeContext from './ThemeContext';
+import { getTheme, ThemeMode } from './theme';
 
-const ThemeProvider: React.FC = ({ children }) => (
-  <EmotionThemeProvider theme={theme}>
-    <MuiThemeProvider theme={theme}>{children}</MuiThemeProvider>
-  </EmotionThemeProvider>
-);
+const ThemeProvider: React.FC = ({ children }) => {
+  const [isDarkMode, setIsDarkMode] = useState();
```

Comment:
If users want to have dark mode by default, how we can achieve that right now? I think this might be the entry point using `UI_OPTIONS` as we did with language, by default would be the default theme.

Reply:
done 😉